data.sql

```diff
@@ -36,3 +36,134 @@ UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Agumon'),
+  (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+  '22-Jul-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Gabumon'),
+  (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+  '2-Feb-2021'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Pikachu'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '5-Jan-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Pikachu'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '8-Mar-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Pikachu'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '14-May-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Devimon'),
+  (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+  '4-May-2021'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Charmander'),
+  (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+  '24-Feb-2021'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Plantmon'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '21-Dec-2021'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Plantmon'),
+  (SELECT id FROM vets WHERE name = 'William Tatcher'),
+  '10-Aug-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Plantmon'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '7-Apr-2021'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Squirtle'),
+  (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+  '29-Sep-2019'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Angemon'),
+  (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+  '3-Oct-2020'
+);
+
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Angemon'),
+  (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+  '4-Nov-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Boarmon'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '24-Jan-2019'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Boarmon'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '15-May-2019'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Boarmon'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '27-Feb-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Boarmon'),
+  (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+  '03-Aug-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Blossom'),
+  (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+  '24-May-2020'
+);
+INSERT INTO visits VALUES (
+  (SELECT id FROM animals WHERE name = 'Blossom'),
+  (SELECT id FROM vets WHERE name = 'William Tatcher'),
+  '11-Jan-2021'
+);
+
```

Comment:
![Screenshot from 2022-09-15 16-50-01](https://user-images.githubusercontent.com/93039359/190396444-ad6bd4a9-b489-4264-89c2-69ab3e692d92.png)

- [ ] Please add the images of your queries result in your pull request summary because it is a requirement in your project requirements page :arrow_up:

Reply:
@HAMMAS-SALEEM The screenshots are in the screenshots folder of the repo.

Reply:
Hey @dodoburner :wave:

You are supposed to add screenshots in the pull request summary. I know that you have them in your repo but it is a requirement in your project requirement to add those images in pull request description so because of that please add your screenshots in your pull request summary :+1:

You can see this :arrow_down: pull request summary to get an idea what i am saying
https://github.com/Fourteen98/vet-clinic/pull/4

Reply:
Okay but no other reviewer required me to add them to the pull request up until now. @HAMMAS-SALEEM

Reply:
@dodoburner
In each milestone, you should put a different set of screenshots. if you put all of them in a folder, it can be really hard for us to find which screenshot belongs to which query